internal/testutil/pkgdata.go

```diff
@@ -1,152 +1,536 @@
 package testutil
 
 import (
-	"encoding/base64"
+	"archive/tar"
+	"bytes"
+	"embed"
+	"path/filepath"
+	"time"
+
+	"github.com/blakesmith/ar"
+	"github.com/klauspost/compress/zstd"
 )
 
 var PackageData = map[string][]byte{}
 
 func init() {
-	baseFilesData, err := base64.StdEncoding.DecodeString(baseFilesBase64)
+	for name, entries := range pkgs {
+		for i, _ := range entries {
+			if !entries[i].noFixup {
+				fixupEntry(&entries[i])
+			}
+		}
+		data, err := buildDeb(entries)
+		if err != nil {
+			panic(err)
+		}
+		PackageData[name] = data
+	}
+}
+
+type tarEntry struct {
+	header  tar.Header
+	noFixup bool
+	content []byte
+}
+
+var pkgs = map[string][]tarEntry{
+	"base-files": {
```

Comment:
if there a way we could dynamically generate this `tarEntry` structure from an existing DEB? 

my thinking here is: while I agree it is a must to have this default, hardcoded DEB for testing, it would also be useful if we could somehow dynamically add other DEBs to this `map`, at test time